runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigurationManager.java

```diff
@@ -98,7 +102,8 @@ public ConfigurationManager(
 
     public NamespaceConfig parse(
         URL configURL,
-        String configText)
+        String configText,
+        Consumer<BindingConfig> writeBindingInfo)
```

Comment:
Let's do this from the call site in the engine since this metadata is owned by the engine.
We also need to figure out how to handle running out of bindings space after several reconfigures.

Reply:
OK